warp10/src/main/java/io/warp10/script/functions/TOHHCODE.java

```diff
@@ -72,13 +69,13 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
     }
 
     if (this.tostring) {
-      String hhcode = Hex.encodeHexString(Longs.toByteArray(geoxppoint));
-    
+      String hhcode = Long.toHexString(geoxppoint);
```

Comment:
This is not equivalent, Long.toHexString may produce strings shorter than 16 bytes.